code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -231,3 +267,156 @@
 		qdel(src)
 		return TRUE
 	return ..()
+
+/* IDLE ANIMATION THING */
+
+/mob/living/simple_animal/mouse/proc/do_idle_animation(anim, duration = 2 SECONDS)
+	ADD_TRAIT(src, TRAIT_IMMOBILIZED, "mouse_animation_trait_[anim]")
+	flick("mouse_[mouse_color]_[anim]",src)
+	addtimer(CALLBACK(src, PROC_REF(animation_end), anim), duration)
+
+/mob/living/simple_animal/mouse/proc/animation_end(anim)
+	REMOVE_TRAIT(src, TRAIT_IMMOBILIZED, "mouse_animation_trait_[anim]")
+
+/mob/living/simple_animal/mouse/proc/sniff()
+	set name = "Sniff"
+	set desc = "Sniff around"
+	set category = "Mouse"
+
+	emote("msniff", intentional = TRUE)
+
+/mob/living/simple_animal/mouse/proc/shake()
+	set name = "Shake"
+	set desc = "Shake yourself"
+	set category = "Mouse"
+
+	emote("mshake", intentional = TRUE)
+
+/mob/living/simple_animal/mouse/proc/scratch()
+	set name = "Scratch"
+	set desc = "Scratch yourself"
+	set category = "Mouse"
+
+	emote("mscratch", intentional = TRUE)
+
+/mob/living/simple_animal/mouse/proc/washup()
+	set name = "Wash Up"
+	set desc = "Wash up yourself"
+	set category = "Mouse"
+
+	emote("mwashup", intentional = TRUE)
+
+/mob/living/simple_animal/mouse/proc/flower()
+	set name = "Show a Flower"
+	set desc = "Show a flower"
+	set category = "Mouse"
+
+	emote("mshowflower", intentional = TRUE)
+
+/mob/living/simple_animal/mouse/proc/smoke()
+	set name = "Smoke"
+	set desc = "Smoke a cigarette"
+	set category = "Mouse"
+
+	emote("msmoke", intentional = TRUE)
+
+/mob/living/simple_animal/mouse/proc/thumb_up()
+	set name = "Thumb Up"
+	set desc = "Give a thumb up"
+	set category = "Mouse"
+
+	emote("mthumbup", intentional = TRUE)
+
+/mob/living/simple_animal/mouse/proc/dance()
+	set name = "Dance"
+	set desc = "Dance with a pair of maracas"
+	set category = "Mouse"
+
+	emote("mdance", intentional = TRUE)
+
+/mob/living/simple_animal/mouse/proc/shakeass()
+	set name = "Shake Ass"
+	set desc = "Shake your ass"
+	set category = "Mouse"
+
+	emote("mshakeass", intentional = TRUE)
+
+/datum/emote/living/simple_animal/mouse/idle
```

Comment:
If you add these emotes, they should go into a proper file just for emotes, which I think mice might already have?